src/middlewares/jwtAuth.ts

```diff
@@ -11,13 +14,37 @@ const generateAccessToken = async (userId: string, userEmail: string) => {
     };
 
     const token = await jwt.sign(payload, JWT_SECRET, {
-      expiresIn: '15days',
+      expiresIn: '1',
     });
 
     return token;
-  } catch (error) {
-    throw new Error('Error generating access token');
+  } catch (error: any) {
+    throw new JwtError(error.message, error.status);
   }
 };
 
-export { generateAccessToken };
+const verifyAccessToken = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<any> => {
+  try {
+    const access_token = await req.cookies.access_token;
+    console.log(access_token);
+    if (!access_token || access_token === undefined || access_token === null) {
```

Comment:
!access_token works for all the condition you are checking, they are falsy values
```suggestion
    if (!access_token) {
```

Reply:
done